Azure/azure-cli
# [AKS] az aks update: add --enable-aad argument

**Description**  


This PR is to port https://github.com/Azure/azure-cli-extensions/pull/1898/files from azure-cli-extension to azure-cli, which supports  updating an existing AAD-Integrated cluster to the new AKS-managed AAD experience.
 
**Testing Guide**  


1. Create a AAD-Integrated cluster: https://docs.microsoft.com/en-us/azure/aks/azure-ad-integration-cli.
2. Update the cluster to the new AKS-managed AAD experience.
    ```
    az aks update -g MyResourceGroup -n MyManagedCluster --enable-aad
    ```
---

This checklist is used to make sure that common guidelines for a pull request are followed.
